cpp/ql/src/Likely Bugs/Leap Year/LeapYear.qhelp

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>The Gregorian Calendar, which has become the internationally accepted civil calendar, the leap year rule is: Every year that is exactly divisible by four is a leap year, except for years that are exactly divisible by 100, but these centurial years are leap years if they are exactly divisible by 400.</p>
+<p>A leap year bug occurs when software (in any language) is written without consideration of leap year logic, or with flawed logic to calculate leap years; which typically results in incorrect results.</p>
+<p>The impact of these bugs may range from almost unnoticeable bugs such as an incorrect date, to severe bugs that affect reliability, availability or even the security of the affected system.</p>
+</overview>
+
+<references>
+<li>U.S. Naval Observatory Website - <a href="https://aa.usno.navy.mil/faq/docs/calendars.php"> Introduction to Calendars</a></li>
+<li>Wikipedia - <a href="https://en.wikipedia.org/wiki/Leap_year_bug"> Leap year bug</a> </li>
+<li>Microsoft Azure blog - <a href="https://azure.microsoft.com/en-us/blog/is-your-code-ready-for-the-leap-year/"> Is your code ready for the leap year?</a> </li>
+</references>
```

Comment:
The `qhelp-preview` job failed with `[ERROR] ql/cpp/ql/src/Likely Bugs/Leap Year/LeapYear.qhelp:16:1: XML document structures must start and end within the same entity.`.  I believe it wants you to add a `</qhelp>` tag here.

Reply:
How can we test the qhelp files to avoid these errors in the future?

Reply:
I personally tend to just rely on the `qhelp-preview` job on the pull request, but I can see that an offline tool might be helpful if you're writing a lot of help.

@semmledocs-ac what do you recommend?